src/bootstrap/defaults/config.user.toml

```diff
@@ -11,3 +11,6 @@ extended = true
 [llvm]
 # Most users installing from source want to build all parts of the project from source, not just rustc itself.
 download-ci-llvm = false
+# We don't want to enable all the llvm backends by default for user profile.
+targets = "host"
+experimental-targets = ""
```

Comment:
I wanted to make it real default actually, but according to

> People who do build LLVM because they do related work want to build everything, including all targets and tools.

I thought it would be better to update this default in profile level.

Reply:
You can make this the new global default, and make it build everything in the `codegen` profile.

Reply:
I don't understand the point of this change if we're only enabling it for people building from source.

Reply:
I thought default profiles are mostly used by new contributors for help. That's why I did this change.

Reply:
> You can make this the new global default, and make it build everything in the `codegen` profile.

I guess that would make more sense, what do you think ? @jyn514

Reply:
The default profiles are used by most contributors, even seasoned ones.

Reply:
I was thinking that bootstrap could autogenerate this list from the list of the targets of rustc.

Reply:
> I was thinking that bootstrap could autogenerate this list from the list of the targets of rustc.

This will require `llvm` to rebuild when rustc targets changed

Reply:
This could be overridden, but I would prefer to have the correct targets if I'm building a cross-compiling compiler than having a broken one, or spend time building useless things.

Reply:
Maybe build llvm with single target if rustc target is equal to host and build complete llvm otherwise? Let's find a proper solution in zulip or different place later. I will revert this change to not block this PR with it.